resources/views/components/user/profile-meta.blade.php

```diff
@@ -37,6 +43,10 @@
     @endif
 </div>
 
+@if ($amIModerator)
+    <x-user.profile.moderation-tools :targetUser="$user" />
+@endif
```

Comment:
The existing Admin button (and related code) still exists lower in the page.

Reply:
Whoops, removed in latest. Thanks.